foreign_cc/cmake.bzl

```diff
@@ -73,6 +62,47 @@ def _create_configure_script(configureParameters):
     flags = get_flags_info(ctx, "<TARGET>")
     no_toolchain_file = ctx.attr.cache_entries.get("CMAKE_TOOLCHAIN_FILE") or not ctx.attr.generate_crosstool_file
 
+    cmake_commands = []
+
+    # If the legacy `make_commands` attribute was not set, use the new
+    # `targets` api for building our target.
+    if not ctx.attr.make_commands:
+        data = ctx.attr.data + getattr(ctx.attr, "tools_deps", [])
+        configuration = "Debug" if is_debug_mode(ctx) else "Release"
+
+        # Generate a list of arguments for cmake's build command
+        build_args = " ".join([
+            ctx.expand_location(arg, data)
+            for arg in ctx.attr.build_args
+        ])
+
+        # Generate commands for all the targets, ensuring there's
+        # always at least 1 call to the default target.
+        for target in ctx.attr.targets or [""]:
+            # Note that even though directory is always passed, the
+            # following arguments can take precedence.
+            cmake_commands.append("{cmake} --build {dir} --config {config} {args} {target}".format(
```

Comment:
Do we need to explicitly choose CMP0061 here? https://cmake.org/cmake/help/latest/policy/CMP0061.html#policy:CMP0061

Reply:
Why would we want to ignore errors from the build commands?

Reply:
My bad, I'd inverted the logic of what CMP0061 is setting.